clang/test/SemaSYCL/intel-fpga-uses-global-work-offset.cpp

```diff
@@ -0,0 +1,49 @@
+// RUN: %clang_cc1 -Wno-return-type -fsycl-is-device -fcxx-exceptions -fsyntax-only -ast-dump -verify -pedantic %s | FileCheck %s
```

Comment:
What about host device?

Reply:
I think there is no need to have host device check as it simply reports "attribute ignored".
You can see that tests for device features mostly don't have host device check (e.g. intel_fpga_local, intel_fpga_loops)

Reply:
If so, can we exclude `SYCLIsHost `from `let LangOpts = [SYCLIsDevice, SYCLIsHost];` ?